.github/workflows/pull-request-review.yml

```
@@ -0,0 +1,46 @@
+            
+            // Filter reviews that have a user login containing "coderabbit" (case-insensitive)
+            const codeRabbitReviews = reviews.filter(review => 
+              review.user.login.toLowerCase().includes('coderabbit')
+            );
+            
+            // Fail if no CodeRabbit reviews are found
+            if (codeRabbitReviews.length === 0) {
+              core.setFailed('ERROR: CodeRabbit has not reviewed this PR.');
+              return;
+            }
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)